source/User_Guide/Marketing_Campaigns/Managing_Contacts/segmenting_your_contacts.md

```diff
@@ -93,12 +112,13 @@ Depending on which condition is selected, the operators you can choose from diff
 </table>
 
 <br/>
-You can select `AND` or `OR` as options, where `AND` requires both conditions to be true and `OR` requires either condition to be true. Once you’ve selected the Condition and the Operator, you can layer additional conditions in the segment to narrow the results to a more targeted audience. 
+When using more than one condition to segment your contacts, you can select `AND` or `OR` as options, where `AND` requires both conditions to be true and `OR` requires either condition to be true. Once you’ve selected the Condition and the Criteria, you can layer additional conditions in the segment to narrow the results to a more targeted audience. 
  
 {% info %}
 You can add up to 15 different conditions per segment.
 {% endinfo %}
 
+![]({{root_url}}/img/Segmentation.png "Segmentation conditions and criteria")
```

Comment:
This looks so much better! I love your flow chart. I would just add it higher up. I think it should be before the table (around line 58) because I think it's more logical to have the conditions listed before you dive into each of the criteria.